manifests/master/agent_tarball.pp

```diff
@@ -9,7 +9,7 @@
   # https://pm.puppetlabs.com/puppet-agent/2015.2.0/1.2.2/repos/puppet-agent-el-6-x86_64.tar.gz
   $filename    = "puppet-agent-${::platform_tag}.tar.gz"
   $download    = "https://pm.puppetlabs.com/puppet-agent/${pe_server_version}/${aio_agent_version}/repos/${filename}"
-  $destination = "${publicdir}/${pe_server_version}/${aio_agent_version}/repos"
+  $destination = "${publicdir}/classroom/${pe_server_version}/${aio_agent_version}/repos"
```

Comment:
This was modified to use the ${publicdir} from the params, with the PS1 files it was trying to put them in the classroom directory that did not exist in fundamentals but does for architect.